### ISSUE TITLE
\ifluatex error

### ISSUE THREAD

**Issue by csgillespie** [yihui/tinytex#128]
**Overview**: 

- The file below works on my local machine
- The file below worked on rocker/verse two days ago.  
- xelatex'ing the file on rocker/verse now raise the following error:

```
! Undefined control sequence.
l.53 \ifluatex
              \else 

Error: Failed to compile main.tex. See https://yihui.name/tinytex/r/#debugging for debugging tips. See main.log for more info.
Execution halted
```

**Reproduce:**
```
docker container run -e PASSWORD=test -e ROOT=TRUE -p 8787:8787 rocker/verse
```
Open the browser on localhost:8787 and log into RStudio. Create a file containing
```
---
title: "YYY"
author: "XXX"
output:
  bookdown::tufte_book2:
    latex_engine: xelatex
---

test
```
and `knit`

---

By filing an issue to this repo, I promise that

- [ X] I have fully read the issue guide at https://yihui.name/issue/.
- [X] I have provided the necessary information about my issue.
 - [X ] I have learned the Github Markdown syntax, and formatted my issue correctly.

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by yihui** [comment]
I don't use Docker myself, so I don't have an idea. Cc @cboettig to see if he has an idea.

**Comment by csgillespie** [comment]
Here is the actual [Dockerfile](https://github.com/jr-packages/jrNotes/blob/master/Dockerfile)

As I mentioned, it worked last week, works with 3.5 pinned. Just not with the latest.

**Comment by yihui** [comment]
`\ifluatex` comes from the **ifluatex** package, which has been included in the default TinyTeX: https://github.com/yihui/tinytex/blob/b54b5889eac8c4be8bfb0139f0ba645a29cafee9/tools/pkgs-custom.txt#L21

Perhaps a certain LaTeX package was broken and forgot to `\usepackage{ifluatex}` [like this](https://github.com/rstudio/rmarkdown/blob/7b18786487a4798959dda959e1f68fa1baeae9a6/inst/rmd/latex/default.tex#L12)? I don't know.

**Comment by csgillespie** [comment]
Yep, that was my thoughts. The latex file generated from the Rmd file is identical using verse:3.5 and verse:latest - so it's not that.

I'll give it a week or two and try again.

**Comment by csgillespie** [comment]
I've just tried rocker/verse:3.6, the issue still persists.

I've moved to rocker/tidyverse and used the ubuntu packages listed at [https://hub.docker.com/r/rocker/verse/dockerfile](verse) +  `texlive` +  `texlive-xetex` +  `texlive-generic-recommended`. I didn't install any tinytex stuff. My .tex files now build.

I don't know enough about latex to make a guess about what is happening.

I did try to install `texlive` on `verse` but ran into compatibility issues.
---

Closing as unable to provide a solution. But hopefully the above is helpful for anyone in the future.
